values-zh-rHK/strings.xml

```diff
@@ -3019,19 +3019,22 @@
     <string name="draft_boulders00_text">風格各異嘅巨大岩石。</string>
     <string name="draft_highway_sign00_title">高速路標志</string>
     <string name="draft_highway_sign00_text">高速路標志用於引導於指示駕駛員嘅行駛方向，注意事項。</string>
-    <string name="draft_road05_ramp00_title">高速路匝道（單向）</string>
-    <string name="draft_road05_ramp00_text">又稱引道，一般系立交/相鄰輔助道路與高速路鏈接嘅路段，具有銜接作用，輔助和引導車輛進入主路</string>
+    <string name="draft_road05_ramp00_title">高速路匝道</string>
+    <string name="draft_road05_ramp00_text">又稱引道，一般系立交/相鄰輔助道路與高速路鏈接嘅路段\n
+具有銜接作用，輔助和引導車輛進入主路</string>
```

Comment:
Remind me to check whether adding an actual newline in a translation xml affects something negatively.